### PR TITLE
Add Overkiz State, Command, CommandParameter enums

### PR DESCRIPTION
**Pull request by iMicknl**

### CODE REVIEW

**Review by tetienne — COMMENTED** [review]
This file starts to be too big. Can you add this in another module?

**Comment by iMicknl** [comment]
> This file starts to be too big. Can you add this in another module?

I was looking into that indeed, how to more easily split this in multiple files. I would like to keep the import simple, instead of having to import from a specific file.

**Comment by iMicknl** [comment]
@tetienne would you know if we can easily change the import to have it from `.enum` instead of `.enum.command`? Or should we just settle for new paths.

**Comment by tetienne** [comment]
> @tetienne would you know if we can easily change the import to have it from `.enum` instead of `.enum.command`? Or should we just settle for new paths.

Perhaps adding the import within the `__init__.py` file. But, like this it does not hurt me.

File: pyhoma/enums/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# flake8: noqa: F401, F403
+
+from .command import *
+from .execution import *
+from .gateway import *
+from .general import *
+from .state import *
+from .ui import *
```
Comment (tetienne) [inline pyhoma/enums/__init__.py:8]: So it was useful finally?

File: pyhoma/enums/ui.py [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+    ZWAVE_DANFOSS_RSLINK = "ZWaveDanfossRSLink"
+    ZWAVE_DOOR_LOCK_CONFIGURATION = "ZWaveDoorLockConfiguration"
+    ZWAVE_FIBARO_ROLLER_SHUTTER_CONFIGURATION = "ZWaveFibaroRollerShutterConfiguration"
+    ZWAVE_HEATIT_THERMOSTAT_CONFIGURATION = "ZWaveHeatitThermostatConfiguration"
+    ZWAVE_NODON_CONFIGURATION = "ZWaveNodonConfiguration"
+    ZWAVE_QUBINO_CONFIGURATION = "ZWaveQubinoConfiguration"
+    ZWAVE_SEDEVICE_CONFIGURATION = "ZWaveSEDeviceConfiguration"
+    ZWAVE_TRANSCEIVER = "ZWaveTransceiver"
+    ZIGBEE_NETWORK = "ZigbeeNetwork"
+    ZIGBEE_STACK = "ZigbeeStack"
```
Comment (tetienne) [inline pyhoma/enums/ui.py:396]: That’s insane. How can they maintain all these widgets?

**Review by tetienne — APPROVED** [review]

File: pyhoma/enums/__init__.py (hunk 1 above)
Comment (iMicknl) [reply pyhoma/enums/__init__.py:8]: The only reason why I did this is to have everything as an import from `enums` (which is also non breaking :D).

File: pyhoma/enums/ui.py (hunk 2 above)
Comment (iMicknl) [reply pyhoma/enums/ui.py:396]: I don't know haha, but they were already added to the enums. Didn't see that before :o.

**Pull request merged**